web/main.rb

```diff
@@ -4,31 +4,38 @@
 require 'string-utility'
 require_relative '../utils/utilities'
 
-gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
-demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
-adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
+$gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
+$demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
+$adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
```

Comment:
@Strikingwolf Yeah, as locals I couldn't access them from the new "analyze" method... what would the best alternative be here according to you?